src/components/PeoplePicker/PeoplePicker.ts

```diff
@@ -22,7 +22,9 @@ namespace fabric {
     private _peoplePickerSearchBox: Element;
     private _peoplePickerResults: NodeListOf<Element>;
     private _isContextualMenuOpen: Boolean;
-    private _selectedPeople: NodeListOf<Element>;
+    private _selectedPeople: Element;
+    private _selectedCount: Element;
+    private _selectedPlural: HTMLElement;
```

Comment:
Is there a good reason why this needs to be an `HTMLElement` vs plain `Element`?